src/renderer/views/account/CreateAccountView.tsx

```diff
@@ -34,11 +34,20 @@ const CreateAccountView = observer(
         password
       );
 
-      const privateKey: PrivateKey = ipcRenderer.sendSync(
-        "unprotect-private-key",
-        address,
-        password
-      );
+      const unprotectedPrivateKey: [
+        string | undefined,
+        any
+      ] = ipcRenderer.sendSync("unprotect-private-key", address, password);
+      if (
+        typeof unprotectedPrivateKey[0] !== "string" ||
+        unprotectedPrivateKey[0] === ""
+      ) {
+        // FIXME: Show a new error page or retry page to complete the account creation.
+        console.error("Failed to unprotect private key");
+        return;
+      }
+
+      const privateKey: string = unprotectedPrivateKey[0];
```

Comment:
How about using [`Destructuring assignment`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)? And it passes also error object like Go does. So how about checking error with it? Like...

```suggestion
      const [privateKey, error]: [
        string | undefined,
        Error | undefined
      ] = ipcRenderer.sendSync("unprotect-private-key", address, password);
      if (
        error !== undefinded ||
        typeof unprotectedPrivateKey[0] !== "string" ||
        unprotectedPrivateKey[0] === ""
      ) {
        // FIXME: Show a new error page or retry page to complete the account creation.
        console.error("Failed to unprotect private key");
        return;
      }
```

Reply:
@moreal I fixed it!